<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ethernet</class>
 <widget class="QDialog" name="Ethernet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ETHERNET SETTING</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbMaol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>MAC</string>
   </property>
   <widget class="QLineEdit" name="leMac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbWm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>DHCP</string>
   </property>
   <widget class="QRadioButton" name="rbYes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbEnter">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>271</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>271</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
